src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -30,7 +35,7 @@ object ConsumptionPlan : Plan(
                 "data" to HandleConnection(
                     StorageKeyParser.parse("reference-mode://{ramdisk://Thing}{ramdisk:///handle/my-handle-id}"),
                     HandleMode.Read,
-                    null,
+                    EntityType(Reader_Data_Spec.schema),
```

Comment:
I believe this is either .schema() or .SCHEMA now? The lowercase schema appears gone.

Reply:
I commented in chat about this... then totally forgot for my PR :)

Reply:
Thanks, Ray.